src/core/env/dev/terraform.tfvars

```diff
@@ -168,6 +168,7 @@ ecommerce_ingress_hostname = "weudev.ecommerce.internal.dev.platform.pagopa.it"
 
 ecommerce_xpay_psps_list = "XPAY"
 ecommerce_vpos_psps_list = "BIC36019,PSPtest1,CHARITY_AMEX,CHARITY_IDPSPFNZ,CHARITY_ISP,40000000001,DINERS,BCITITMM,MARIOGAM,73473473473,60000000001,PAYTITM1,POSTE1,ProvaCDI,50000000001,IDPSPFNZ,70000000001,10000000001,idPsp2,irraggiungibile_wisp,prova_ila_1,pspStress50,pspStress79,pspStress80,pspStress81"
+ecommerce_npg_psps_list  = ""
```

Comment:
Shold we remove this modifications into core domain?

Reply:
You're right!